Xamarin.Essentials/Sms/Sms.shared.cs

```diff
@@ -18,18 +20,14 @@ public static Task ComposeAsync(SmsMessage message)
 
     public class SmsMessage
     {
-        public SmsMessage()
-        {
-        }
-
-        public SmsMessage(string body, string recipient)
+        public SmsMessage(string body, string[] recipients)
```

Comment:
We can also make this an `IEnumerable<string>`

Reply:
Done